orderer/common/cluster/util_test.go

```diff
@@ -470,7 +539,32 @@ func TestEndpointconfigFromConfigBlockFailures(t *testing.T) {
 }
 
 func TestBlockValidationPolicyVerifier(t *testing.T) {
+	dir := filepath.Join(os.TempDir())
+	os.Mkdir(dir, 0o700)
+	defer os.RemoveAll(dir)
```

Comment:
Maybe this RemoveAll() is removing temp files that the other tests need?

Reply:
I think that is it.... it should be `t.TempDir()` ... 
Thanks!